screens/main/about.js

```diff
@@ -10,18 +10,17 @@ import {
   Image,
   Platform,
 } from 'react-native';
-import {Button, InfoBox} from '../../components';
+import {Button, InfoBox, PageTitle} from '../../components';
 import {Colors, GlobalStyles} from '../../styles';
 
 export default function InfoScreen({navigation}) {
 
   return (
     <SafeAreaView style={styles.container}>
       <ScrollView>
-        <View style={styles.content}>
-          <Text style={GlobalStyles.h1}>iWalk Info</Text>
+        <View>
+          <PageTitle title="iWalk Information" />
```

Comment:
Since we need to localize this app, could you pull your strings into a separate file like `strings.json` and down the road, we'll swap out which file we import for display?

Reply:
There's a separate story for localization that we can assign to @drumwolf for this if @johnsonphan95 is no longer working on it. It will end up touching almost every file, so we should do it in a different branch.